topo_processor/cog/create_cog.py

```diff
@@ -0,0 +1,22 @@
+import os
+from typing import List
+
+from topo_processor.cog.command import Command
+from topo_processor.cog.execution import ExecutionDocker
+
+
+async def create_cog(input_file: str, output_dir: str, volumes: List[str], compression_method: str):
```

Comment:
the `volumes` are `input_file` and `output_dir` and should not need another list

`compression_method` is not actually used in the cog creation